### PR TITLE
docs: fix broken anchor links 

### PR DESCRIPTION
**Pull request by mldangelo**
- Fix broken links in LLM vulnerability documentation
- Update express and related dependencies
- Upgrade prism-react-renderer and typescript
- Add onBrokenAnchors warning in docusaurus config
- Adjust formatting in vulnerability types table

### CODE REVIEW

File: site/docs/red-team/plugins/prompt-extraction.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ By incorporating the Prompt Extraction plugin in your LLM red teaming strategy,
 ## Related Concepts
 
 - [Information Disclosure](../llm-vulnerability-types.md#privacy-and-security)
-- [Social Engineering](../llm-vulnerability-types.md#social-engineering)
-- [Model Inversion Attacks](../llm-vulnerability-types.md#model-inversion)
+- [Social Engineering](../llm-vulnerability-types.md#misinformation-and-misuse)
+- [Model Inversion Attacks](../llm-vulnerability-types.md#technical-vulnerabilities)
```
Comment (typpo) [inline site/docs/red-team/plugins/prompt-extraction.md]: All 3 of these are hallucinated, let's completely remove them

**Review by typpo — APPROVED** [review]
thanks!

File: site/docs/red-team/llm-vulnerability-types.md [hunk 2]
```diff
@@ -9,17 +9,17 @@ This page documents categories of potential LLM vulnerabilities and failure mode
...
   - [Criminal Activities](#criminal-activities)
   - [Harmful Content](#harmful-content)
   - [Misinformation and Misuse](#misinformation-and-misuse)
 - [Common LLM vulnerabilities by application type](#common-llm-vulnerabilities-by-application-type)
 
-Each vulnerability type is supported by Promptfoo's open-source LLM red teaming tool, with the `Plugin` column corresponding to the plugin ID in the tool. [Learn more](/docs/red-team/quickstart/).
+Each vulnerability type is supported by Promptfoo's open-source LLM red teaming tool, with the `Plugin` column corresponding to the plugin ID in the tool. [Learn more](/docs/red-team/quickstart).
 
-## Privacy and Security
+### Privacy and Security
```
Comment (mldangelo) [inline site/docs/red-team/llm-vulnerability-types.md]:
```suggestion
## Privacy and Security
```

File: site/docs/getting-started.md [hunk 3]
```diff
@@ -289,9 +289,9 @@ A simple `npx promptfoo@latest eval` will run the example. Also note that you ca
 
 Produces this HTML table:
 
-![Side-by-side evaluation of LLM model quality, gpt3 vs gpt4, html output](https://user-images.githubusercontent.com/310310/235490527-e0c31f40-00a0-493a-8afc-8ed6322bb5ca.png)
+![Side-by-side evaluation of LLM model quality, gpt4o vs gpt4o-mini, html output](https://user-images.githubusercontent.com/310310/235490527-e0c31f40-00a0-493a-8afc-8ed6322bb5ca.png)
```
Comment (mldangelo) [inline site/docs/getting-started.md]:
```suggestion
![Side-by-side evaluation of LLM model quality, gpt-4o vs gpt-4o-mini, html output](https://user-images.githubusercontent.com/310310/235490527-e0c31f40-00a0-493a-8afc-8ed6322bb5ca.png)
```

File: README.md [hunk 4]
```diff
@@ -233,15 +233,15 @@ You can also output a nice [spreadsheet](https://docs.google.com/spreadsheets/d/
...
 
 ```
-npx promptfoo eval -p prompts.txt -r openai:gpt-3.5-turbo openai:gpt-4 -o output.html
+npx promptfoo eval -p prompts.txt -r openai:gpt-4o openai:gpt-4o-mini -o output.html
 ```
 
 Produces this HTML table:
 
-![Side-by-side evaluation of LLM model quality, gpt3 vs gpt4, html output](https://user-images.githubusercontent.com/310310/235490527-e0c31f40-00a0-493a-8afc-8ed6322bb5ca.png)
+![Side-by-side evaluation of LLM model quality, gpt4o vs gpt4o-mini, html output](https://user-images.githubusercontent.com/310310/235490527-e0c31f40-00a0-493a-8afc-8ed6322bb5ca.png)
```
Comment (mldangelo) [inline README.md]:
```suggestion
![Side-by-side evaluation of LLM model quality, gpt-4o vs gpt-4o-mini, html output](https://user-images.githubusercontent.com/310310/235490527-e0c31f40-00a0-493a-8afc-8ed6322bb5ca.png)
```

**Pull request merged**